libafl_targets/src/sancov_pcguard.rs

```diff
@@ -19,12 +19,12 @@ pub unsafe extern "C" fn __sanitizer_cov_trace_pc_guard(guard: *mut u32) {
     let pos = *guard as usize;
     #[cfg(feature = "sancov_pcguard_edges")]
     {
-        *EDGES_MAP.get_unchecked_mut(pos) = 1;
+        (EDGES_MAP_PTR as *mut u8).add(pos).write_volatile(1);
```

Comment:
Are you sure this needs to be volatile? The process will exit at the end, so everything should have been written. I fear it could decrease perf a bit, otherwise?

Reply:
not needed 😅